packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -149,7 +154,7 @@ export const getItemClassNames = memoizeFunction(
         {
           flexBasis: '0',
           padding: '0 8px',
-          minWidth: 28
+          minWidth: CONTEXTUAL_SPLIT_MENU_MINWIDTH
```

Comment:
I wonder why we use `minWidth` in here? Can this split menu grow in size? Maybe it should be fixed because I think with the the new `width` style you are adding to the `splitPrimary` it might never grow beyond 28px.

Reply:
Thanks for fixing the git history issue 👍

Reply:
In reality I don't I'm hard coding the rest of the control to 100% - 28px, so the split button will always be 28px. I was just trying to have less code change than more, but I'm open to changing it to width instead.

Reply:
The menu button should grow to fill the space available, I think. The width on the split primary seems to ensure that it will always try to an additional 28 pixels to fit the split menu button. Should there be a minimum width for the contextualmenu maybe? I don't know if we can assume that. This seems like a good middle ground.